clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/CopybookDownloaderForDsn.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+import { ProfileUtils } from "../../util/ProfileUtils";
+import { CopybookName } from "../CopybookDownloadService";
+import { CopybookDownloader } from "./CopybookDownloader";
+import { DownloadUtil } from "./DownloadUtil";
+import { ZoweExplorerDownloader } from "./ZoweExplorerDownloader";
+
+/**
+ * Copybook downloader from MVS using Zowe Explorer
+ */
+export class CopybookDownloaderForDsn
+  extends ZoweExplorerDownloader
+  implements CopybookDownloader
+{
+  constructor(
+    storagePath: string,
+    explorerAPI: IApiRegisterClient | undefined,
```

Comment:
```suggestion
    explorerAPI: IApiRegisterClient,
```
and the same change in all the sisters, super class and in DownloadStrategyResolver.

Reply:
CopybookDownloader is removed now

Reply:
updated